tests/test_license.py

```diff
@@ -86,6 +86,7 @@ def setUp(self):
         ]
 
         skippedDirectoryPaths = [
+            'src/opendr/perception/pose_estimation/hr_pose_estimation/algorithm',
```

Comment:
This will probably need to be removed if the corresponding directory is removed.